juriscraper/pacer/iquery.py

```diff
@@ -0,0 +1,187 @@
+import pprint
+import sys
+
+from .docket_report import BaseDocketReport
+from .reports import BaseReport
+# from .utils import clean_pacer_object
+from ..lib.log_tools import make_default_logger
+from ..lib.string_utils import clean_string, harmonize, \
+    force_unicode
+
+logger = make_default_logger()
+
+
+class iQueryReport(BaseDocketReport, BaseReport):
```

Comment:
For the class and file name, I think we should go with a more semantic wording. iQuery meant nothing to me before this PR because I don't pay a ton of attention to the URLs. I'm not sure what to call this page, but I've tried (and probably failed) to go with semantic names, not PACER-implementation names where possible?

Reply:
Yes. s/iQuery/CaseQuery/gi
and rewriting git history (see later on)